test/integration/test_collections.py

```diff
@@ -317,7 +317,7 @@ def test_get_datasets(conservator):
 
 
 @pytest.mark.skip()
-def test_download_datasets(conservator, tmp_cwd):
+def test_download_datasets(conservator):
```

Comment:
Why is the tmp_cwd removed?  This is supposed to run the test in a temporary directory that is automatically removed after the test completes, so that the downloaded dataset is deleted.

Reply:
`tmp_cwd` was marked as unused, so I removed it.

Reply:
It's a case where the fixture used for its side effect of creating a temporary path and cleaning up afterward, so tmp_cwd is not used directly (if the unused parameter is causing a linter complaint, one option is to put the line `del tmp_cwd` as the first line of the test code, similar to putting `(void)(param)` in C code, e.g. for callbacks that always receive fixed parameters in a callback that doesn't use all of them):
```@pytest.fixture()
def tmp_cwd(tmp_path):
    """
    Set the current working directory to a temporary path for the duration
    of the test. Then restore the current working directory and delete the path.
    """
    cwd = os.getcwd()
    os.chdir(tmp_path)
    yield tmp_path
    os.chdir(cwd)
    shutil.rmtree(tmp_path)```

Reply:
I updated the code to use `@pytest.mark.usefixtures("tmp_cwd")` instead of passing `tmp_cwd` as an unused arg.